### DISCUSSION TITLE
iOS and Android support?

### DISCUSSION THREAD

**Discussion by alexey-leonovich** [git-ecosystem/git-credential-manager#1739] [Q&A]
Hello.
Is it possible to use GCM on iOS and Android?
What credential stores are supported?
Thank you.

**Comment by dscho** [comment]
Are you sure you want to use Git on iOS or Android?

**Reply by alexey-leonovich** [reply]
I want to use GCM to store credentials (via [CredentialManager](https://github.com/devlooped/CredentialManager) wrapper) in my C# application targeting Windows\Linux\MacOS\iOS\Android.

**Reply by dscho** [reply]
That's a bit of a different use case than GCM caters for, though. Maybe you can leverage the code in a fork?

**Reply by alexey-leonovich** [reply]
I'm afraid I don't have enough programming skills to add iOS\Android support in a fork.
P. S. I've searched a bit and found that [WorkingCopy](https://apps.apple.com/ua/app/working-copy-git-client/id896694807) and [ Git Drive](https://apps.apple.com/us/app/gitdrive-git-client-server/id1215974700) applications exist for iOS. All Git client applications were removed from Google Play for some reason. git itself in written with C, GCM is written with C# - so it is definitely a doable task to add iOS\Android support.
